crates/bifrost/Cargo.toml

```diff
@@ -71,7 +82,11 @@ criterion = { workspace = true, features = ["async_tokio"] }
 enumset = { workspace = true }
 googletest = { workspace = true }
 paste = { workspace = true }
-pprof = { version = "0.14", features = ["criterion", "flamegraph", "frame-pointer"] }
+pprof = { version = "0.14", features = [
+    "criterion",
+    "flamegraph",
+    "frame-pointer",
+] }
```

Comment:
The problem with this formatting is that it makes it harder to re-sort the dependency list alphabetically. I'd appreciate if we keep a single line per dep.